aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -12,6 +12,51 @@
 from aws_lambda_powertools.utilities.typing import LambdaContext
 
 logger = logging.getLogger(__name__)
+APPLICATION_JSON = "application/json"
```

Comment:
Because we're adding APPLICATION_JSON as part of this PR, what are your thoughts in creating a `content_types.py` should customers ask for binary mime types, for example?

That'd be easier to reason for newcomers and less options in the autocomplete that matters (subjective but sure).

Reply:
@heitorlessa same as the exceptions, i wonder if we should then make `api_gateway` into a "real" module?

Reply:
Exceptions is different. Could you expand on what "real" module means? Do you mean a separate utility altogether, a separate package in a different namespace, a framework?

Reply:
@heitorlessa  just mean a sub folder for api gatway.